node/node.go

```diff
@@ -737,6 +743,11 @@ func (node *Node) handleNewChainHeads(ctx context.Context, prevHead types.TipSet
 				if err := node.StorageMiner.OnNewHeaviestTipSet(newHead); err != nil {
 					log.Error(err)
 				}
+				if node.StorageFaultSlasher != nil {
```

Comment:
This shouldn't be nested in the StorageMiner condition. StorageFaultSlasher doesn't depend on StorageMiner not being nil. StorageMiner != nil should imply that StorageFaultSlash != nil, so this condition could go away, but my preference would be to have botth conditions side by side.